geometry_msgs/PointStamped pixel
---
geometry_msgs/PointStamped pixel3d
